Makefile

```diff
@@ -1,7 +1,7 @@
 # Don't edit Makefile! Use conf-* for configuration.
 
 SHELL=/bin/sh
-NROFF?=nroff
+NROFF=nroff
```

Comment:
It would be awesome if you could add to the comment fixing this a "Fixes: <sha1>" line with a reference to the commit that broke it. Same for the "make -C" ones.